js/libs/builder.js

```diff
@@ -86,6 +86,7 @@ function generateFormDefaults(data) {
     onSubmit: [],
     template: false,
     saveProgress: true,
+    autobindProfileEditing: false,
```

Comment:
why is this required?

Reply:
It was actually a mistake in not having it. We use this in some places